models/klaviyo__events.sql

```diff
@@ -6,7 +6,7 @@
             "field": "occurred_on",
             "data_type": "date"
         } if target.type == 'bigquery' else none,
-        incremental_strategy = 'merge' if target.type != 'snowflake' else 'delete+insert',
+        incremental_strategy = 'merge' if target.type not in ('snowflake', 'postgres', 'redshift') else 'delete+insert',
```

Comment:
Should the other models not merge with snowflake, or is it just specific this one that snowflake can't do merge?

Reply:
if i remember correctly it was just this model (i have no idea why though)